File: FE/DATN-Fashion-Shop-FE/src/app/dto/StorePaymentRequest.ts
export interface StorePaymentRequest  {
  userId: number | null;
  storeId: number;
  couponId?: number | null;
  paymentMethodId: number;
  totalPrice: number;
  totalAmount: number;
  taxAmount: number;
  transactionCode: string | null;
  payUrl?: string;
}

File: DATN-Fashion-Shop-BE/src/main/java/com/example/DATN_Fashion_Shop_BE/dto/request/store/StorePaymentRequest.java
package com.example.DATN_Fashion_Shop_BE.dto.request.store;

import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Data
@Builder
@Getter
@Setter
public class StorePaymentRequest {
    private Long userId;
    private Long storeId;
    private Long couponId;
    private Long paymentMethodId;
    private String transactionCode;
    private Double totalAmount;
    private Double totalPrice;
    private Double taxAmount;
}
